filesystem/src/main/java/com/dropbox/android/external/fs3/RecordPersister.kt

```diff
@@ -2,28 +2,28 @@ package com.dropbox.android.external.fs3
 
 import com.dropbox.android.external.fs3.filesystem.FileSystem
 import com.dropbox.android.external.store4.legacy.BarCode
-import java.util.concurrent.TimeUnit
 import javax.inject.Inject
```

Comment:
I think we can get rid of these `@Inject` annotations and the `javax.inject:javax.inject` artifact.

Reply:
Right, Inject is not very multiplatform. Removed it.